packages/SidePanel/src/components/LockBodyScroll.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+function LockBodyScroll() {
```

Comment:
Not a fan of repeating this file in every component. Perhaps something we could add into `helpers`?

Reply:
let's merge this one, and maybe @DanilAgafonov can consolidate these two into the helpers in his PR.